webservices/load_legal_docs.py

```diff
@@ -368,6 +371,33 @@ def get_citations(data):
             raise Exception("Could not parse citation")
     return {"us_code": us_codes, "regulations": regulations}
 
+def map_pre2012_citation(title, section, archived_mur_citation_map={}):
```

Comment:
There is already a function that does this for current MURs - https://github.com/18F/openFEC/blob/develop/webservices/load_current_murs.py#L154. I suggest we refactor and use it here.